lib/spack/llnl/util/cpu/microarchitectures.json

```diff
@@ -1093,7 +1096,7 @@
         "clang": {
           "versions": "3.9:",
           "family": "ppc64le",
-          "name": "power8",
+          "name": "pwr8",
```

Comment:
As far as I can tell, clang treats "power8" and "pwr8" as synonyms, but LLVM (and, transitively, Rust) only recognizes "pwr8", so this change prefers the abbreviated version. Somebody with a better understanding of LLVM and microarchitectures may want to take a look, though.